src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web.UnitTests/Validators/ClarificationPageValidatorTests.cs

```diff
@@ -129,5 +131,59 @@ public async Task When_clarification_response_word_count_exceeds_maximum_then_an
             Assert.AreEqual("Clarification response must be 300 words or less", response.Errors.First().ErrorMessage);
             Assert.AreEqual("ClarificationResponse", response.Errors.First().Field);
         }
+
+        [Test]
+        public async Task When_FilesToUpload_has_file_that_exceeds_maximum_filesize_then_an_error_is_returned()
+        {
+            const int maxFileSize = 5 * 1048576;
```

Comment:
can we convert1048576 to a meangingful variable name :)

Reply:
done